core/src/main/java/de/unistuttgart/iaas/amyassist/amy/core/speech/TextToSpeech.java

```diff
@@ -46,7 +47,8 @@
  * 
  * @author Tim Neumann, Kai Menzel
  */
-public class TextToSpeech implements VoiceOutput {
+@Service(TextToSpeech.class)
```

Comment:
additional the service must register in core https://github.com/AmyAssist/Amy/pull/133/files#diff-1b37aa06bea959779ddee594afd733b7L169. This is required because speech is not a plugin